src/libcore/cmp.rs

```diff
@@ -8,17 +8,16 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-//! Functionality for ordering and comparison.
+//! Functionality for ordering and relational operations.
 //!
-//! This module defines both [`PartialOrd`] and [`PartialEq`] traits which are used
-//! by the compiler to implement comparison operators. Rust programs may
-//! implement [`PartialOrd`] to overload the `<`, `<=`, `>`, and `>=` operators,
-//! and may implement [`PartialEq`] to overload the `==` and `!=` operators.
+//! This module defines four traits related to ordering and comparisons:
 //!
-//! [`PartialOrd`]: trait.PartialOrd.html
-//! [`PartialEq`]: trait.PartialEq.html
+//! * [`PartialEq`]: which overloads the `==` and `!=` operators,
+//! * [`PartialOrd`] (requires `PartialEq`): which overloads the `<`, `<=`, `>`, 
+//!   and `>=` operators,
 //!
-//! # Examples
+//! state that values of a type can be compared for equality, or ordered, but that
```

Comment:
I find this hard to parse, where does the sentence even begin and what is the subject matching this verb?